.changelog/1604.changed.txt

```diff
@@ -0,0 +1 @@
+feat(ConfigProvider): Modify config merge to do replace instead of update for Specific tags
```

Comment:
```suggestion
feat(ConfigProvider)!: Modify config merge to do replace instead of update for Specific tags
```

and also please rename file to `1604.breaking.txt`